servant-waargonaut.cabal

```diff
@@ -36,15 +41,15 @@ library
   exposed-modules:     Servant.API.ContentTypes.Waargonaut
   -- other-modules:
   -- other-extensions:
-  build-depends:       base        >= 4.11   && < 4.13
-                     , servant     >= 0.14.1 && < 0.16
-                     , waargonaut  >= 0.5    && < 0.6
-                     , attoparsec
-                     , bytestring
-                     , text
-                     , http-media
-                     , lens
-                     , wl-pprint-annotated
+  build-depends:       base                >= 4.11   && < 4.13
+                     , servant             >= 0.14.1 && < 0.16
+                     , waargonaut          >= 0.5    && < 0.6
+                     , attoparsec          >= 0.13.2 && < 0.14
+                     , bytestring          >= 0.10.8 && < 0.11
+                     , text                >= 1.2.3 && < 1.3
+                     , http-media          >= 0.7.1 && < 0.8
+                     , lens                >= 4.16.1 && < 4.17
```

Comment:
This can be bumped to lens < 4.17 here and in the test suite

Reply:
I erm, *ahem*, removed most of the bounds for the moment to see if it build on everything I was expecting it to. I'll need to move them back in.